CHANGELOG.md

```diff
@@ -0,0 +1,581 @@
+# Changelog
+
+## v2.0.3 (01/01/1970)
+- [**closed**] normalize signature line endings before loading signature block to xml-crypto [#512](https://github.com/node-saml/passport-saml/pull/512)
```

Comment:
Do we have to have the status here? Some of these status and tags really add nothing to the change log IMO.

Reply:
Maybe we can use this config value to remove the tags: https://github-tools.github.io/github-release-notes/options.html#ignore-labels

Reply:
Can the date format be ISO 8601 standard? E.g. 2021-01-07

Reply:
it will require custom template with date parsing/formatting code, there is no option for date format

Reply:
It seems that way, sad... thankfully, someone already wrote the template, and all we have to do is paste it in :)

https://github.com/github-tools/github-release-notes/issues/147#issuecomment-584759005

Reply:
![image](https://user-images.githubusercontent.com/75169/103930433-69785400-5127-11eb-93bf-f0822cd88fa2.png)